flink-cdc-base/src/main/java/com/ververica/cdc/connectors/base/config/BaseSourceConfig.java

```diff
@@ -87,6 +90,11 @@ public boolean isIncludeSchemaChanges() {
         return includeSchemaChanges;
     }
 
+    @Override
+    public boolean isScanNewlyAddedTableEnabled() {
+        return scanNewlyAddedTableEnabled;
+    }
+
```

Comment:
Are we missing the `scan.newly-added-table.enabled` config option for table api?

Reply:
Hmm, I don't see the Table API implementation in the `flink-cdc-base` module... Am I missing something?

Reply:
Maybe we need to pass the `scan.newly-added-table.enabled` option via `OracleTableSourceFactory` and `MongoDBTableSourceFactory`.

Reply:
Well, I think actually enabling the option to be usable in connectors implementing the Incremental Snapshotting framework should be in a different pull request. Partially because of the scope of the change, but also because it might require connector-specific changes. And it definitely requires a lot of connector-specific testing.